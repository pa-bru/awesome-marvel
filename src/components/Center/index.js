import React from 'react'
import PropTypes from 'prop-types'

const Center = ({ children }) => (
  <div
    style={{
      display: 'flex',
      justifyContent: 'center',
      alignItems: 'center',
      width: '100%',
    }}
  >
    {children}
  </div>
)

Center.propTypes = {
  children: PropTypes.node.isRequired,
}

export default Center
