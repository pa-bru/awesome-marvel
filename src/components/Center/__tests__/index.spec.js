import React from 'react'
import { shallow } from 'enzyme'
import Center from '../'

describe('Center', () => {
  it('should render correctly', () => {
    // When
    const tree = shallow(<Center>COUCOU</Center>)

    // Then
    expect(tree).toMatchSnapshot()
  })
})
