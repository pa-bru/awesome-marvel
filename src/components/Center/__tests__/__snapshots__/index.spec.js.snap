// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Center should render correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  COUCOU
</div>
`;
